### DISCUSSION TITLE
.NET Aspire

### DISCUSSION THREAD

**Discussion by adambajguz** [ravendb/ravendb#19464] [Ideas]
Hi, have you thought about adding .NET Aspire support?

https://github.com/dotnet/aspire/discussions/2119
https://github.com/CommunityToolkit/Aspire/discussions/104

**Comment by ppekrol** [comment]
cc @shiranshalom 

hi @adambajguz 

We are currently researching this topic. You can track the progress here: https://issues.hibernatingrhinos.com/issue/RavenDB-23016/Add-.NET-Aspire-Integration-for-RavenDB

**Reply by adambajguz** [reply]
great, thanks!

**Comment by shiranshalom** [comment]
@adambajguz 
We have completed the integration of RavenDB with .NET Aspire, and the NuGet packages are now available! 🎉

You can find them here:
- [CommunityToolkit.Aspire.RavenDB.Client (9.1.1-beta.173)](https://www.nuget.org/packages/CommunityToolkit.Aspire.RavenDB.Client/9.1.1-beta.173)
- [CommunityToolkit.Aspire.Hosting.RavenDB (9.1.1-beta.173)](https://www.nuget.org/packages/CommunityToolkit.Aspire.Hosting.RavenDB/9.1.1-beta.173)

These packages enable seamless integration of RavenDB within .NET Aspire applications. 
If you have any feedback or run into any issues, feel free to share!

**Reply by adambajguz** [reply]
wow, awoesome, great to hear that !!!
